e3/vcs/git.py

```diff
@@ -63,6 +64,29 @@ def __init__(self, working_tree):
         """
         self.working_tree = working_tree
 
+    @classmethod
+    def create(cls, repo_path, initial_content_path=None):
+        """Create a local Git repository.
+
+        :param repo_path: a local directory that contains the repository
+        :type repo_path: str
+        :param initial_content_path: directory containing the initial content
+            of the repository. If set to None an empty repository is created.
+        :type initial_content_path: str
+        :return: the URL of the newly created repository
+        :rtype: str
+        """
+        repo_path = os.path.abspath(repo_path)
+        repo = GitRepository(repo_path)
```

Comment:
Could we use "cls" instead of hardcoding "GitRepository" here? That way, if we happen to change the class' name, we wouldn't have to change it here.